<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTC_OpenCloseTrack</class>
 <widget class="QDialog" name="CTC_OpenCloseTrack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open or Close Track</string>
  </property>
  <widget class="QComboBox" name="LineNameBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Green</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="BlockNumberBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LineLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </widget>
  <widget class="QLabel" name="BlockNumLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Block Number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangeTrackButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Track</string>
   </property>
  </widget>
  <widget class="QComboBox" name="OpenCloseBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Open</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Closed</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>New status</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
